examples/structured_1d_dgsem/elixir_burgers_PERK3.jl

```diff
@@ -0,0 +1,53 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the (inviscid) Burgers' equation
+
+equations = InviscidBurgersEquation1D()
+
+initial_condition = initial_condition_convergence_test
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+solver = DGSEM(polydeg = 4, surface_flux = flux_lax_friedrichs)
```

Comment:
Maybe we use indeed p=3 as this is the standard degree for most examples.

In that case, you can maybe increase `CFL` below.